code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4476,12 +4512,35 @@
 	name = "guac dip"
 	desc = "A recreation of the ancient Sol 'Guacamole' dip using tofu, limes, and spices. This recreation obviously leaves out mole meat."
 	nachotrans = /obj/item/reagent_containers/food/snacks/chip/nacho/guac
+	avahtrans = /obj/item/reagent_containers/food/snacks/chip/miniavah/guac
 	chiptrans = /obj/item/reagent_containers/food/snacks/chip/guac
 	icon_state = "dip_guac"
 	reagents_to_add = list(/singleton/reagent/nutriment = 20)
 	reagent_data = list(/singleton/reagent/nutriment = list("guacmole" = 20))
 	filling_color = "#35961D"
 
+/obj/item/reagent_containers/food/snacks/dip/sarmikhir
+	name = "sarmikhir"
+	desc = "A traditional Tajaran cream made of fermented Fatshouter milk. Traditionally used with bread or as condiment."
```

Comment:
```suggestion
	desc = "A traditional tajaran cream made of fermented Fatshouter milk. Traditionally used with bread or as condiment."
```

Reply:
wiki says otherwise.

Reply:
following wiki formatting, going to keep it capitalised